scripts/generate_index.py

```diff
@@ -0,0 +1,29 @@
+import os
+from pathlib import Path
+source = Path(__file__).resolve().parent.parent / "source"
+source = source.resolve()
+info = {}
+for p in Path(source).rglob("*"):
+    if p.suffix not in ['.md']:
+        continue
+    relpath = os.path.relpath(p.parent, start=source)
```

Comment:
NIT: We could use `"*.md"` reduce the amount of paths to iterate. Thoughts?
```suggestion
for p in Path(source).rglob("*.md"):
    relpath = os.path.relpath(p.parent, start=source)
```

Reply:
Done